livereload/middleware.py

```diff
@@ -19,29 +19,27 @@ class LiveReloadScript(MiddlewareMixin):
     """
 
     def process_response(self, request, response):
-        content_type = response.get('Content-Type', '').split(';')[0].strip().lower()
-        if (not settings.DEBUG or
-                content_type not in ['text/html', 'application/xhtml+xml'] or
-                not hasattr(response, 'content')):
+        content_type = response.get("Content-Type", "").split(";")[0].strip().lower()
+        if (
+            not settings.DEBUG
+            or content_type not in ["text/html", "application/xhtml+xml"]
+            or not hasattr(response, "content")
+        ):
             return response
 
-        soup = BeautifulSoup(
-            smart_str(response.content),
-            'html.parser',
+        content = smart_str(response.content)
+        insertion_point = "</head>"
+        livereload_script_tag = format_html(
+            """<script src="{}:{}/livereload.js"></script>""",
```

Comment:
@kbehlers You're going to want `http` here:

```
src="http://{}:{}/livereload.js"></script>"""
```

Reply:
Just rounding back to this, thank you. Fixed ✅ .